flank-scripts/build.gradle.kts

```diff
@@ -28,7 +28,7 @@ shadowJar.apply {
     }
 }
 // <breaking change>.<feature added>.<fix/minor change>
-version = "1.1.5"
+version = "1.1.6"
```

Comment:
I would go with `1.2.0` because you adding new features

Reply:
You're right, thanks!